yapapi/services/service.py

```diff
@@ -365,20 +361,19 @@ async def reset(self) -> None:
         Handlers of a restarted service are called more then once - all of the cleanup necessary between calls
         should be implemented here. E.g. if we initialize a counter in :func:`Service.__init__` and increment it
         in :func:`Service.start()`, we might want to reset it here to the initial value.
-
-        Target implementation (0.10.0 and up) will raise NotImplementedError. Current implementation only warns about
```

Comment:
We should mention that the default implementation is a no-op

Reply:
plus, we should mention the relationship between the `reset` here and the `restart_condition` below

Reply:
Done